README.md

```diff
@@ -180,3 +181,9 @@ https://github.com/deveronuas/sssdk.git
 ```
 
 For more details, see [Adding Package Dependencies to Your App](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app).
+
+## Release History
```

Comment:
No need for this, just add `Release Process`

Release history is maintained in the Github releases https://github.com/deveronuas/sssdk/releases you can just add a link to it